src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -61,6 +65,9 @@ public Users findOne(@PathVariable Long id) {
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
     public Users create(@RequestBody Users user) {
+
+        user.setPassword(passwordEncoder.encode(user.getPassword()));
```

Comment:
This is ok, but the card requires that you have a different resource to update the password from a user, different from the resource where you are creating/updating them.

Reply:
Yes! There should be another endpoint with the purpose of updating the password alone :)